net/gensync/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            nislab gensync 1.0.4 v
+git.branch              vanilla
```

Comment:
Is it necessary to manually set the branch to `vanilla`?  If you do not set this, MacPorts will just use the `v1.0.4` git tag, which is usually what's expected.

Reply:
`git.banch` has now been removed. Thanks